### ISSUE TITLE
`show_masks`, and `show_anns` did not work in batch mode

### ISSUE THREAD

**Issue by thipokKub** [opengeos/segment-geospatial#42]
### Environment Information

-   samgeo version: 0.5.0
-   Python version: 3.10.11
-   Operating System: Ubuntu (?). I run in colab

### Description
I run the example notebook in colab (GPU) with cuda device. Sam can output in batch mode when generate when set `batch=True`
```
sam.generate(image, output=mask,, foreground=True, unique=True, batch=True
```
However this is not compatible with `show_masks`, and `show_anns` as in the example

### What I Did

I ran the code. For `show_masks` it output
```
No masks found. Please run generate() first.
```
and for `show_anns` it output
```
Please run generate() first.
```

**Comment by giswqs** [comment]
Yes, those two functions do not support batch mode yet. Batch mode is for large image only
